chainindexer.go

```diff
@@ -400,9 +400,11 @@ out:
 		case indexJob := <-workChan:
 			addrIndex, err := a.indexBlockAddrs(indexJob.blk)
 			if err != nil {
-				adxrLog.Errorf("Unable to index transactions of"+
-					" block: %v", err)
-				a.server.Stop()
```

Comment:
The other critical shutdowns are triggered by distinct error conditions. I think the server shutdown should be left in place, but within this new "are we shutting down" conditional. 

Additionally, this "are we already shutting down" guard of error logging should be added to the other areas within the `chainindexer`that may trigger a critical shutdown. The extraneous error messages are harmless, but may cause users to conclude that an attempt at a clean shutdown failed.